File: 0782-jewels-and-stones/0782-jewels-and-stones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    let count=0;
    let Jset=new Set(jewels);
    for (let char of stones){
        if(Jset.has(char)){
           count++
        }
    }
    return count
};

File: 0782-jewels-and-stones/0782-jewels-and-stones.py
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        """
        :type jewels: str
        :type stones: str
        :rtype: int
        """
        count=0
        for char in jewels:
            for charec in stones:
                if char==charec:
                    count+=1
        return count
        